src/config/ThemeProvider.tsx

```diff
@@ -54,7 +54,10 @@ export default class ThemeProvider extends React.Component<
 
   static getDerivedStateFromProps(props, state) {
     const { useDark } = props;
-    if (useDark !== state.useDark) {
+    const isTheme = (theme) => {
+      return !(Object.keys(theme).length === 0 && theme.constructor === Object);
```

Comment:
I'm not sure I understand why this function? And I don't know what `isTheme` is even supposed to mean. What is this function trying to look for? That the theme is present? I thought we were supposed to look for whether the `theme` object changed or not.